app/ethercat/unit/ethercat_t.cpp

```diff
@@ -0,0 +1,70 @@
+#include "gtest/gtest.h"
+
+#include "Fibre.hpp"
+#include "DataModel.hpp"
+
+#include "SOES_hooks.h"
+
+extern void fill_input_buffer(_Rbuffer& buffer);
+
+bool isConfigured;
+uint8_t data;
+
+TEST(EtherCATTest, test_fill_buffer)
```

Comment:
this test will change soon, because i am working on making the ethercat.cpp more generic